sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/implementation/EntityHelperTest.java

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus.implementation;
+
+import com.azure.messaging.servicebus.models.CreateQueueOptions;
+import com.azure.messaging.servicebus.models.EntityStatus;
+import com.azure.messaging.servicebus.models.QueueProperties;
+import org.junit.jupiter.api.Test;
+
+import java.time.Duration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * {@link EntityHelper} tests.
+ */
+class EntityHelperTest {
+    @Test
+    void createQueue() {
+        // Arrange
+        final String queueName = "some-queue";
+        final CreateQueueOptions expected = new CreateQueueOptions(queueName)
+            .setAutoDeleteOnIdle(Duration.ofSeconds(15))
+            .setDefaultMessageTimeToLive(Duration.ofSeconds(50))
+            .setDeadLetteringOnMessageExpiration(true)
+            .setDuplicateDetectionHistoryTimeWindow(Duration.ofSeconds(13))
+            .setEnableBatchedOperations(false)
+            .setEnablePartitioning(true)
+            .setForwardTo("Forward-To-This-Queue")
+            .setForwardDeadLetteredMessagesTo("Dead-Lettered-Forward-To")
+            .setLockDuration(Duration.ofSeconds(120))
+            .setMaxDeliveryCount(15)
+            .setMaxSizeInMegabytes(2048)
```

Comment:
If we set negative values to these numbers, We should validate these values.

Reply:
Our guidelines say to do as little validation as possible and offset it to the service. I'd expect the service to do this validation and throw.